### ISSUE TITLE
How can I get ShapeNet-Part dataset?

### ISSUE THREAD

**Issue by tangsankou** [Julie-tang00/Point-BERT#75]
I referenced the link #https://github.com/Julie-tang00/Point-BERT/blob/master/DATASET.md to download the ShapeNet-Part dataset, but I found that the website [https://shapenet.cs.stanford.edu/media/shapenetcore_partanno_segmentation_benchmark_v0_normal.zip] is not accessible. Could you please advise how I can obtain the ShapeNet-Part dataset?

**Comment by wuxiaofei01** [comment]
> I referenced the link #https://github.com/Julie-tang00/Point-BERT/blob/master/DATASET.md to download the ShapeNet-Part dataset, but I found that the website #https://shapenet.cs.stanford.edu/media/shapenetcore_partanno_segmentation_benchmark_v0_normal.zip is not accessible. Could you please advise how I can obtain the ShapeNet-Part dataset?

do you have some solution？ i have the same question

**Comment by tangsankou** [comment]
> > I referenced the link #https://github.com/Julie-tang00/Point-BERT/blob/master/DATASET.md to download the ShapeNet-Part dataset, but I found that the website #https://shapenet.cs.stanford.edu/media/shapenetcore_partanno_segmentation_benchmark_v0_normal.zip is not accessible. Could you please advise how I can obtain the ShapeNet-Part dataset?
> 
> do you have some solution？ i have the same question

No, I found this website https://huggingface.co/datasets/ShapeNet/ShapeNetCore through https://www.shapenet.org/download/shapenetcore and tried to download, but I have not been receiving the confirmation email.

**Comment by wuxiaofei01** [comment]
> > > I referenced the link #https://github.com/Julie-tang00/Point-BERT/blob/master/DATASET.md to download the ShapeNet-Part dataset, but I found that the website #https://shapenet.cs.stanford.edu/media/shapenetcore_partanno_segmentation_benchmark_v0_normal.zip is not accessible. Could you please advise how I can obtain the ShapeNet-Part dataset?
> > 
> > 
> > do you have some solution？ i have the same question
> 
> No, I found this website https://huggingface.co/datasets/ShapeNet/ShapeNetCore through https://www.shapenet.org/download/shapenetcore and tried to download, but I have not been receiving the confirmation email.

thank you for your reply, i give up...

**Comment by ywyue** [comment]
FYI: the [PointCloudMamba](https://github.com/SkyworkAI/PointCloudMamba/tree/main/data) repo contains a GoogleDrive [link](https://drive.google.com/uc?id=1W3SEE-dY1sxvlECcOwWSDYemwHEUbJIS) to download the ShapeNetPart data.

**Comment by Fazel-AVB** [comment]
After downloading the ShapeNet separately, is it still possible to use `from torch_geometric.datasets import ShapeNet`? I have downloaded it via Huggingface; however, to use ShapeNet, it seems that it needs a specific `.zip` file which includes pre-processed text files (e.g., `points.txt`, `seg_label.txt`). Nevertheless, this raw file from Huggingface has the format of `.obj` 3D model files!

**Comment by durden609** [comment]
Do you have some solution now？
